### ISSUE TITLE
The free membership won't work. What I do is, go up to the search bar, click on it, and type in javascript: then I paste in the code, and nothing happens. by the way I'm on a school computer and I can't inspect the prodigy website to put in the code that way. I hope I can get a response soon.

### ISSUE THREAD

**Issue by 0779850** [rxzyx/prodigy-hack#112]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by James12346543** [comment]
@rxzyx 
I agree with this it wont work and i am also on a school computer and its not workinggggggggg can you please help us

**Comment by IUseBlooketHacks** [comment]
Make a bookmark, change the link to javascript: and paste your code after the javascript and click on the bookmark on the prodigy tab.

**Comment by James12346543** [comment]
Still didn't work
